### PR TITLE
Set JAVA_VENDOR define

### PR DESCRIPTION
**Pull request by pshipton**
Update the OpenJ9 default for VENDOR_URL_BUG and override the default
for VENDOR_URL_VM_BUG.

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

File: closed/OpenJ9.gmk [hunk 1]
```diff
@@ -286,6 +286,7 @@ stage-j9 :
 	$(call openj9_copy_tree,$(OUTPUTDIR)/vm/omr,$(OPENJ9OMR_TOPDIR))
 
 OPENJ9_VERSION_VARS := \
+	COMPANY_NAME \
```
Comment (keithc-ca) [inline closed/OpenJ9.gmk:289]: Why is `VENDOR_URL` not in this list (like jdk8, jdk11)?

File: closed/openj9_version_info.h.in [hunk 2]
```diff
@@ -37,5 +37,6 @@
 #define OPENJ9_TAG                "@OPENJ9_TAG@"
 #define J9JDK_EXT_VERSION         "@J9JDK_EXT_VERSION@"
 #define J9JDK_EXT_NAME            "@J9JDK_EXT_NAME@"
+#define JAVA_VENDOR               "@COMPANY_NAME@"
```
Comment (keithc-ca) [inline closed/openj9_version_info.h.in:40]: Why is this not added like jdk8, jdk11?
```
#define JAVA_VENDOR_URL "@VENDOR_URL@"
```

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: closed/openj9_version_info.h.in (hunk 2 above)
Comment (pshipton) [reply closed/openj9_version_info.h.in:40]: It's related to the OpenJ9 code that uses JAVA_VENDOR_URL, it's only used up to Java 11. https://github.com/eclipse/openj9/blob/master/runtime/vm/vmprops.c#L611-L639

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
Jenkins compile xlinux jdk15 depends eclipse/openj9#11216

**Comment by keithc-ca** [comment]
Jenkins compile xlinux jdk15 depends eclipse/openj9#11216

**Pull request merged**